config/routes.rb

```diff
@@ -41,4 +44,10 @@
   end
 
   root 'home#index'
+
+  namespace :api, defaults: { format: :json } do
+    as :user do
+      get '/companies', to: 'responses#companies'
```

Comment:
could you start making it resourceful?

Reply:
why responses controller if it is actually companies?

Reply:
thought of a controller to handle responses of api. not to mix it all together.
and it has its own way of authentication to get here